### ISSUE TITLE
commands don't line up with command prompt in transcripts

### ISSUE THREAD

**Issue by andrewschultz** [iftechfoundation/ifcomp#127]
Run a simple game on dev.ifcomp.org. Type in a few commands.

What we see in the HTML:

>

GET UP.X ROBE.TAKE ANALGESIC

What we expect:

>GET UP.X ROBE.TAKE ANALGESIC

No information is lost, but can we format this so that it looks nicer and the author can see more of the transcript at once?

**Comment by erkyrath** [comment]
I think Markdown has borked your comment. You mean

    >
    
    GET UP.X ROBE.TAKE ANALGESIC

Versus:

    >GET UP.X ROBE.TAKE ANALGESIC

Right?

**Comment by andrewschultz** [comment]
Correct. Sorry about that! I edited it to reflect what it should be.
